### PR TITLE
Code Quality Fixes

### PR DESCRIPTION
**Pull request by lakshyacis**
__init__.py pylint fixes
cipher pylint fixes
hash pylint fixes
openssl pylint fixes

### CODE REVIEW

File: src/pyelliptic/cipher.py [hunk 1]
```diff
@@ -44,30 +49,34 @@ def get_all_cipher():
 
     @staticmethod
     def get_blocksize(ciphername):
+        """This Method returns cipher blocksize"""
         cipher = OpenSSL.get_cipher(ciphername)
         return cipher.get_blocksize()
 
     @staticmethod
     def gen_IV(ciphername):
+        """"""
```
Comment (PeterSurda) [inline src/pyelliptic/cipher.py]: `""Generate random initialization vector"""`

File: src/pyelliptic/cipher.py [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
 #  Copyright (C) 2011 Yann GUIBET <yannguibet@gmail.com>
 #  See LICENSE for details.
 
 from openssl import OpenSSL
 
 
-class Cipher:
+# pylint: disable=redefined-builtin
+class Cipher:    # pylint: disable=old-style-class
```
Comment (PeterSurda) [inline src/pyelliptic/cipher.py]: Can you make this a new style class and test if anything breaks? I think that's cleaner.

Comment (lakshyacis) [reply src/pyelliptic/cipher.py]: Okay, will try and let you know.

Comment (lakshyacis) [reply src/pyelliptic/cipher.py]: After making old-style class to new style class. All thinks works fine.Also test sending / receiving a message from a different computer.
So should i change old-style-class to new-style-class ?

Comment (PeterSurda) [reply src/pyelliptic/cipher.py]: Yes please

**Pull request merged**